python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -178,18 +178,23 @@ def get_path_prefix_option(uri):
         return "&path_prefix"
 
 
-def test_separation_between_libraries_with_prefixes(object_storage_uri_incl_bucket):
+@pytest.mark.parametrize(
+    "lib_type",
+    ["moto_s3_uri_incl_bucket", "azurite_azure_uri_incl_bucket"] if AZURE_SUPPORT else ["moto_s3_uri_incl_bucket"],
+)
+def test_separation_between_libraries_with_prefixes(request, lib_type):
```

Comment:
Please revert changes to this test as I have rewritten these test setup in my branch (and it's not relevant to mongo).

Reply:
It will be handled in the final rebase

Reply:
Reverted